### PR TITLE
New TC for [PolicyTableUpdate] OnStatusUpdate(UPDATE_NEEDED) on new P…

### PR DESCRIPTION
**Pull request by istoimenova**
To run the script you need to replace:
user_modules/shared_testcases/testCasesForPolicyTable.lua
user_modules/shared_testcases/testCasesForBuildingSDLPolicyFlag
from #32.
Common functions are under review.

@maria-mih please review the script according to template.

### CODE REVIEW

File: test_scripts/Polices/Policy_Table_Update/007_ATF_P_TC_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request.lua [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+commonFunctions:newTestCasesGroup("Test")
+function Test:TestStep_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request()
+  local correlationId = self.mobileSession:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.appName } })
+  :Do(function(_,data)
+      local hmi_app1_id = self.applications[config.application1.registerAppInterfaceParams.appName]
+      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UPDATE_NEEDED"})
+
+      testCasesForPolicyTableSnapshot:create_PTS(true, {
```
Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/007_ATF_P_TC_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request.lua]: @istoimenova create_PTS should be renamed as verify_PTS

File: test_scripts/Polices/Policy_Table_Update/007_ATF_P_TC_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request.lua [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+require('user_modules/AppTypes')
+local mobile_session = require('mobile_session')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+function Test:Precondition_remove_user_connecttest()
+  os.execute( "rm -f ./user_modules/connecttest_RAI.lua" )
+end
+
+function Test:TestStep_ConnectMobile()
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/007_ATF_P_TC_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request.lua]: Please change prefix to Precondition_

File: test_scripts/Polices/Policy_Table_Update/007_ATF_P_TC_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request.lua [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  self:connectMobile()
+end
+
+function Test:TestStep_StartNewSession()
```
Comment (maria-mih) [inline test_scripts/Polices/Policy_Table_Update/007_ATF_P_TC_OnStatusUpdate_UPDATE_NEEDED_new_PTU_request.lua]: Please change prefix to Precondition_

**Comment by istoimenova** [comment]
@LuxoftAKutsan please merge.

**Pull request merged**